DiagnosticsExtension/Controllers/ConnectionStringValidationController.cs

```diff
@@ -85,5 +115,23 @@ public async Task<HttpResponseMessage> ValidateAppSetting(string appSettingName,
                 return Request.CreateErrorResponse(HttpStatusCode.NotFound, $"AppSetting {appSettingName} not found");
             }
         }
+
+        [HttpGet]
+        [Route("test")]
+        public async Task<IHttpActionResult> Test()
+        {
+            try
+            {
+                MsiValidator msi = new MsiValidator();
+                MsiValidatorInput input = new MsiValidatorInput(ResourceType.KeyVault, "fdc4ce08-1981-464a-a490-1604c022df3f");
```

Comment:
I am assuming this is just some random guid ? Maybe using Guid.NewGuild().ToString() method here

Reply:
Accidentally checked testing code in, will remove it.

Reply:
removed